internal/journald/processentry_test.go

```diff
@@ -125,9 +126,10 @@ func TestEntryProcessing(t *testing.T) {
 		pid      string
 	}
 	tests := []struct {
-		name string
-		args args
-		want *auditevent.AuditEvent
+		name                       string
+		args                       args
+		expectsRemoteUserLoginChan bool
```

Comment:
Anyone have a better name for this variable?

Reply:
looks good to me.